### PR TITLE
Fix search / fix ThreadLocalChange for bot

### PR DESCRIPTION
**Pull request by albertaleksieiev**

### CODE REVIEW

File: src/main/java/songbox/house/util/JsonUtils.java [hunk 1]
```diff
@@ -81,6 +84,30 @@ public static VkSearchResponseDto searchFromNewsFeedResponseToObject(final Strin
     }
 
     public static <T> T responseToObject(final String body, final Class<T> clazz) {
+        // Try #1
+        try {
```
Comment (Denstden) [inline src/main/java/songbox/house/util/JsonUtils.java:99]: Maybe create 2 methods:
1) private static <T> Optional<T> parse(final String body, final Class<T> clazz)
2) private static <T> T parseWithOldApi(final String body, final Class<T> clazz)
And change this public method body to
`return parse(body, clazz).orElseGet(() -> parseWithOldApi(body, clazz));`

With this implementation your comments "Try #1" and "Try Old api" will be redundant. Ideally, the code should be self-documented :)
Anyway, it's just syntax sugar and your solution is cool)

File: src/main/java/songbox/house/util/JsonUtils.java [hunk 2]
```diff
@@ -81,6 +84,30 @@ public static VkSearchResponseDto searchFromNewsFeedResponseToObject(final Strin
...
+            JSONArray payload = new JSONObject(body.substring(body.indexOf(JSON_DELIMITER_V2) + JSON_DELIMITER_V2.length())).getJSONArray("payload");
+            for (int i = 0; i < payload.length(); i++) {
+                Object payloadObj = payload.get(i);
+                if (!(payloadObj instanceof JSONArray)) {
+                    continue;
+                }
+
+                JSONArray jsonArray = (JSONArray)payloadObj;
```
Comment (Denstden) [inline src/main/java/songbox/house/util/JsonUtils.java]: Format the code

**Review by Denstden — APPROVED** [review]
lgtm

File: src/main/java/songbox/house/util/JsonUtils.java (hunk 1 above)
Comment (albertaleksieiev) [reply src/main/java/songbox/house/util/JsonUtils.java:99]: Done 😉

**Pull request closed without merging**